resolwe/flow/executors/docker/__init__.py

```diff
@@ -183,3 +184,7 @@ def end(self):
     def terminate(self, data_id):
         """Terminate a running script."""
         subprocess.call(shlex.split('{} rm -f {}'.format(self.command, data_id)))
+
+    def post_register_hook(self):
+        """Pull Docker images required by processes after the 'register' command is done."""
+        call_command('list_docker_images', pull=True)
```

Comment:
@abukosek, what would a rough estimate of the total size of Docker images downloaded on a deploy? And how long does it typically take to download them?

I'm thinking whether it would be wise to make the pulling configurable when we are testing our deployment scripts in Vagrant VMs.

Reply:
The images used for tests are rather small (less than 1 GB, from memory), so pulling them takes a few minutes at most, but I agree that this should be configurable.  I'll add a settings option for this.